virttest/utils_disk.py

```diff
@@ -422,6 +423,9 @@ def __init__(self, disk):
         self.g = guestfs.GuestFS()
         self.disk = disk
         self.g.add_drive(disk)
+        service = SpecificServiceManager("libvirtd")
+        if not service.status():
+            service.start()
```

Comment:
Need to check whether the service has been really started. And, suggest to rename `service` to `libvirtd`.